lib/active_record/connection_adapters/amazon_timestream/quoting.rb

```diff
@@ -1,13 +1,15 @@
-# frozen_string_literal: true
-
 module ActiveRecord
   module ConnectionAdapters
     module AmazonTimestream
       module Quoting
         def quote_table_name(table_name)
           "\"#{@database}\".\"#{quote_string(table_name)}\""
```

Comment:
Can you reference `quote_table_name_without_database` here so that if we change one, it changes both?

```
"\"#{@database}\".#{quote_table_name_without_database(table_name)}"
```